index.js

```diff
@@ -0,0 +1,185 @@
+// INDEX
+// - Variables
+// - Initialize
+// - Navigation
+// - Step 1
+// - Step 2
+// - Step 3
+
+
+
+const step0 = document.getElementById('step0');
+const step1 = document.getElementById('step1');
+const step2 = document.getElementById('step2');
+const step3 = document.getElementById('step3');
+const tax = 0.08;
+
+let customerBankAccount = 0;
+let customerSpendLimit = 0;
+let phoneChoice = "";
+let accessoryChoice = "";
+let bundleSize = "";
+let totalSpent = 0;
+let costPerBundle = 0;
+
+
+
+initialize();
+
+// *** each step gets its own display component for input and/or info ***
+
+function initialize() {
+  toggleDisplaySteps(step0, step1); //TODO - optional argument not working? had to add 2nd arg
```

Comment:
Probably want to do this last, as it will expose elements that could trigger your onclicks, but your onclicks haven't been set yet. Click handlers should be set up first.

Reply:
Noted, got them rearranged before showing the step.